packages/api/src/routes/helpers/handle-params.ts

```diff
@@ -0,0 +1,10 @@
+import { NextFunction, Request, Response } from "express";
+
+export const handleParams = (req: Request, res: Response, next: NextFunction): void => {
+  req.aggregatedParams = {
```

Comment:
is it possible to have two params with the same key in the same route?

Reply:
like `:id/blah/:id` i dont think that should be possiole anways

Reply:
This will collapse though values, you may want to maintain a list per param in that case... makes the logic a bit more painful

Reply:
I'd check that Express overwrites in this way too anyway -- i.e. at the child, there's only one `req.id` and it's the last one in the route.

Reply:
Yea it will default to the last one